infoschema/tables.go

```diff
@@ -387,7 +387,7 @@ var tablesCols = []columnInfo{
 	{name: "TIDB_ROW_ID_SHARDING_INFO", tp: mysql.TypeVarchar, size: 255},
 	{name: "TIDB_PK_TYPE", tp: mysql.TypeVarchar, size: 64},
 	{name: "TIDB_PLACEMENT_POLICY_NAME", tp: mysql.TypeVarchar, size: 64},
-	{name: "TIDB_DIRECT_PLACEMENT", tp: mysql.TypeVarchar, size: types.UnspecifiedLength},
+	{name: "TIDB_DIRECT_PLACEMENT", tp: mysql.TypeVarchar, size: 512},
```

Comment:
Is 512 the maximum length of Direct Placement? I'm not sure there even is a maximum?

Reply:
It deserves discussing, IMO, the maximum of varchar 16383 is enough, and this table is small, we do not need care the storage overhead too much.

Reply:
I like 512 as a max. We can always increase it later (I might have also suggested 1024), but decreasing it is difficult.